xbmc/cores/AudioEngine/Utils/AEChannelInfo.h

```diff
@@ -32,7 +32,7 @@ class CAEChannelInfo {
 
 public:
   CAEChannelInfo();
-  CAEChannelInfo(const enum AEChannel* rhs);
+  explicit CAEChannelInfo(const enum AEChannel* rhs);
```

Comment:
yes, i see. do you want to rely on implicit casts or should i change the code to be explicit?